### DISCUSSION TITLE
`Build ipa for distribution` exited with status code 65

### DISCUSSION THREAD

**Discussion by Baptmix** [codemagic-ci-cd/codemagic-docs#1585] [Q&A] [answered]
Hello,

ID : 64413102bd727820e40d7a92

I have encountered a problem that prevents building with Codemagic. It is a bug which is adjustable only by modifying the Pods-App-frameworks.sh file by changing source="$(readlink "${source}")" by source="$(readlink -f "${source}")".

The problem is that this bug can't be patched with Codemagic. So the build is failing:

rsync error: some files could not be transferred (code 23) at /AppleInternal/Library/BuildRoots/97f6331a-ba75-11ed-a4bc-863efbbaf80d/Library/Caches/com.apple.xbs/Sources/rsync/rsync/main.c(996) [sender=2.6.9]

The PhaseScriptExecution command failed with a non-zero output code.

![image](https://user-images.githubusercontent.com/56630992/233375928-07366682-b230-46a3-987f-acbe0bf511de.png)

Thanks

**Comment by Baptmix** [comment]
I had also this bug on another project : 644292181eb0eeb90bd3a093

**Comment by montreux** [comment, marked as answer]
In `codemagic.yaml` change the line

    cocoapods: default

to

    cocoapods: 1.12.1

There is a conflict between Xcode 12.3 and Cocoapods 1.12.0, fixed in Cocoapods 1.12.1. Unfortunately the default build environment on codemagic is currently Xcode 12.3 and Cocoapods 1.12.0, so we need to explicitly state the Cocoapods version until the default build environment includes the fixed version of Cocoapods.

**Comment by icarusdust** [comment]
Hi @Baptmix As @montreux mentioned, it is about the conflict between Cocoapods and Xcode 14.3. Either change the version to 1.12.1 or executing `brew update cocoapods` should help you solve it as well but I suggest you to simply change it to 1.12.1 as running another script will add additional time.

**Comment by Baptmix** [comment]
Hi @icarusdust @montreux, all is working fine !

Thanks,
